src/redux/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const SET_APP_DATA = "SET_APP_DATA";
 export const SET_ERROR_ACTION = "SET_ERROR_ACTION";
+export const SET_AUTORIZATION_STATUS_TRUE = "SET_AUTORIZATION_STATUS_TRUE";
```

Comment:
also, you could use enums:
`export enum ActionTypes { 
  SET_APP_DATA = "SET_APP_DATA", 
  SET_ERROR_ACTION = "SET_ERROR_ACTION", 
  SET_AUTORIZATION_STATUS = "SET_AUTORIZATION_STATUS" 
  }`

Reply:
fixed